tests/RadixTree.spec.ts

```diff
@@ -0,0 +1,104 @@
+import { expect } from './helper';
+import RegExpNode from '../src/RegExpNode';
+import BaseNode from '../src/BaseNode';
+import RadixTree from '../src/RadixTree';
+
+describe('RadixTree', () => {
+    let radixTree: RadixTree;
+    let bbb: BaseNode;
+    let cc: BaseNode;
+    let def: BaseNode;
+    let bbbccc: BaseNode;
+    let bbbee: BaseNode;
+    let cceee: BaseNode;
+    let ccfff: BaseNode;
+    let eeeORddd: BaseNode;
+    let twoDigits: BaseNode;
+    let digits: BaseNode;
+    let cceeeggg: BaseNode;
+    let cceeehhhh: BaseNode;
+    let jjj: BaseNode;
+
+    beforeEach(() => {
```

Comment:
It's a large `beforeEach`. I'd consider the refactoring in the future

Reply:
it is just to test find() as we don't have other option to create the tree for now